### PR TITLE
Option

### PR DESCRIPTION
**Pull request by ijackson**
As discussed in #5.  Thanks for your attention :-).

### CODE REVIEW

File: src/option.rs [hunk 1]
```diff
@@ -0,0 +1,46 @@
+//! Module to adaopt `humantime_serde` to `Option`
```
Comment (jean-airoldie) [inline src/option.rs:1]:
```suggestion
//! Convenience module to serialize `Option` via `humantime_serde`.
```

**Review by jean-airoldie — CHANGES_REQUESTED** [review]
Minor rewording to make it clear that its a helper module. Other than this minor thing, looks good to me.

File: src/option.rs (hunk 1 above)
Comment (ijackson) [reply src/option.rs:1]: option: Clarify docs.

**Comment by ijackson** [comment]
Cool. thanks.

**Review by jean-airoldie — APPROVED** [review]

**Comment by jean-airoldie** [comment]
Alright, I'll make a release as well. Thanks!

**Comment by ijackson** [comment]
Thanks :-).  Have switched our codebase to using the newly published version.

**Pull request merged**